app/views/reports/instrument_unavailable_reports/report_table.html.haml

```diff
@@ -0,0 +1,16 @@
+.updated_values{style: "display: none"}
```

Comment:
This looks to duplicate reports/report_table.html.haml Can we share the partial?

If we're not going to share some of the partials, then we don't really need to abide by all the instrument reporter APIs and could probably make the reports much cleaner. I could go either way.